corehq/apps/hqwebapp/static/hqwebapp/js/ui-element.js

```diff
@@ -93,9 +93,9 @@ var uiElement;
             var translated = langcodeTag.translate_delim(value);
             this.ui.find('.lang-text').remove();
             if (translated.lang) {
+                this.ui.css("position", "relative");
                 var langcode_button = langcodeTag.button_tag(
-                    $('<a href="#" class="btn btn-inverse btn-mini lang-text"'
-                      + ' style="color:#ffffff; text-decoration: none;" />'),
+                    $('<a href="#" class="btn btn-info btn-xs btn-mini lang-text" style="position: absolute; top: 6px; right: 6px;" />'),
```

Comment:
I didn't check where all ui element is used, but looking now, does look like it's mostly app manager, with one reference in `corehq/apps/groups/templates/groups/group_members.html` (not sure if that's B2 or B3). I'm inclined to leave it B2-compatible until we're all switched over and then do a final sweep for B2 classes. There are other js files that'll need that kind of cleanup anyway (the multimedia popup comes to mind).